salt/vault/map.jinja

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+{% import_yaml "vault/maps/defaults.yaml" or {} as defaults %}
+{% import_yaml "vault/maps/osfamilymap.yaml" or {} as osfamilymap %}
+{% import_yaml "vault/maps/initfamilymap.yaml" or {} as initfamilymap %}
+
+{%- set merged_defaults = salt['grains.filter_by'](defaults,
+    default='vault',
+    merge=salt['grains.filter_by'](osfamilymap, grain='os_family',
```

Comment:
the alternative syntax is a little cleaner... `salt.grains.filter_by(...)`

can do that throughout this file, instead of the dictionary-key syntax